From 70d895098eaa0de97b6db6d06c990fe82bf59b91 Mon Sep 17 00:00:00 2001
From: Yann Dirson <yann.dirson@vates.tech>
Date: Wed, 6 Nov 2024 10:34:37 +0100
Subject: [PATCH] Fix calloc call

Newer gcc issues a warning for swapped arguments, and we use -Werror

Signed-off-by: Yann Dirson <yann.dirson@vates.tech>
---
 src/arg-list.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/arg-list.c b/src/arg-list.c
index b19f3cc..df39819 100644
--- a/src/arg-list.c
+++ b/src/arg-list.c
@@ -27,7 +27,7 @@
 // =============================================================================
 
 static inline ArgNode *arg_list_create_node () {
-  ArgNode *node = calloc(sizeof *node, 1);
+  ArgNode *node = calloc(1, sizeof *node);
   if (!node)
     syslog(LOG_ERR, "Unable to alloc list node.");
   return node;
-- 
2.39.5

